pyphare/pyphare/cpp/__init__.py

```diff
@@ -1,17 +1,21 @@
 
 
+def cpp_lib():
+    import importlib
+    if not __debug__:
+        return importlib.import_module("pybindlibs.cpp")
+    try:
+        return importlib.import_module("pybindlibs.cpp_dbg")
+    except ImportError as err:
+        return importlib.import_module("pybindlibs.cpp")
```

Comment:
if I understand correctly:, we could run in debug, but for some reason can't improt the debug cpp version then we'd "silently" fallback on the regular one? do we want that to be silent?

Reply:
python doesn't know what mode cmake was built with, this assumes if there's no dbg library, cmake wasn't built in debug

Reply:
what's this variable then __debug__

Reply:
oh that's the python -O probably?

Reply:
"\_\_debug\_\_" is false when exectuting with "python -O script.py"